### PR TITLE
Fixes #35941 - More details in recurring-logic list

### PR DESCRIPTION
**Pull request by adamruzicka**
Everything the UI shows should be obtainable through hammer as well.

Requires https://github.com/theforeman/foreman-tasks/pull/703

### CODE REVIEW

File: lib/hammer_cli_foreman_tasks/recurring_logic.rb [hunk 1]
```diff
@@ -6,9 +6,15 @@ class ListCommand < HammerCLIForeman::ListCommand
       output do
         field :id, _('ID')
         field :cron_line, _('Cron line')
+        field :task_count, _('Task count')
+        field :action, _('Action')
+        field :last_occurence, _('Last occurrence')
+        field :next_occurence, _('Next occurrence')
```
Comment (ofedoren) [inline lib/hammer_cli_foreman_tasks/recurring_logic.rb:12]: For fields with datetime data it's better to use `Fields::Date` formatter.

**Review by ofedoren — COMMENTED** [review]
Thanks, @adamruzicka, tested with the main PR, LGTM.

One suggestion, but it's up to you:

File: lib/hammer_cli_foreman_tasks/recurring_logic.rb (hunk 1 above)
Comment (adamruzicka) [reply lib/hammer_cli_foreman_tasks/recurring_logic.rb]: Good point, done

**Review by ofedoren — APPROVED** [review]
Thanks, @adamruzicka, let's get this in.

**Pull request merged**